.pre-commit-config.yaml

```diff
@@ -0,0 +1,115 @@
+exclude: |
+    (?x)
+    # NOT INSTALLABLE ADDONS
+    # END NOT INSTALLABLE ADDONS
+    # Files and folders generated by bots, to avoid loops
+    ^setup/|/static/description/index\.html$|
+    # We don't want to mess with tool-generated files
+    .svg$|/tests/([^/]+/)?cassettes/|^.copier-answers.yml$|^.github/|
+    # Maybe reactivate this when all README files include prettier ignore tags?
+    ^README\.md$|
+    # Library files can have extraneous formatting (even minimized)
+    /static/(src/)?lib/|
+    # Repos using Sphinx to generate docs don't need prettying
+    ^docs/.*$|
+    # You don't usually want a bot to modify your legal texts
+    (LICENSE.*|COPYING.*)
+default_language_version:
+    python: python3
+    node: "16.17.0"
+repos:
+    - repo: local
+      hooks:
+          # These files are most likely copier diff rejection junks; if found,
+          # review them manually, fix the problem (if needed) and remove them
+          - id: forbidden-files
+            name: forbidden files
+            entry: found forbidden files; remove them
+            language: fail
+            files: "\\.rej$"
+          - id: en-po-files
+            name: en.po files cannot exist
+            entry: found a en.po file
+            language: fail
+            files: '[a-zA-Z0-9_]*/i18n/en\.po$'
+    - repo: https://github.com/oca/maintainer-tools
+      rev: 4cd2b852214dead80822e93e6749b16f2785b2fe
+      hooks:
+          # update the NOT INSTALLABLE ADDONS section above
+          - id: oca-update-pre-commit-excluded-addons
+          - id: oca-fix-manifest-website
+            args: ["https://github.com/OCA/l10n-spain"]
```

Comment:
That value dosn't make sense. But in fact, i think that a lot of checks should be removed, because have no sense in openupgradelib library context.

Reply:
Yes, removed.